addons/mail/static/src/models/activity/activity.js

```diff
@@ -282,7 +295,13 @@ function factory(dependencies) {
          * Do not use this value in a 't-raw' if the activity has been created
          * directly from user input and not from server data as it's not escaped.
          */
-        note: attr(),
+        note: attr({
+            compute: '_computeNote',
+            default: undefined,
```

Comment:
Do you need this line? `undefined` should be the default value by default.

Reply:
Well you are right, I don't know why I did it in the first place.